### ISSUE TITLE
all caps not in diplomatic div

### ISSUE THREAD

**Issue by emylonas** [Brown-University-Library/iip-texts#69]
shouldn't this transcription be in the diplomatic dev? sina0114/ (Gaia checks)

EM looks for all caps in the transcription div when diplomatic is empty.

**Comment by emylonas** [comment]
sina0114 is fine. EM check anyway

**Comment by emylonas** [comment]
possible diplomatic 
caes0156. Also has lots of single `<unclear>` letters that could be `<unclear>` words

**Comment by emylonas** [comment]
This is a uniqued list of capital letters that are in the transcription div. That is each inscription appears only once, although it may have more than 1 capital letter. Most are ok. There are some cases where unclear letters are entered as capitals, even though the rest of the transcription is edited. Possibly worth a look, but not urgent. I'm putting the list here for safekeeping.
```
akko0006.xml: <gap reason="illegible" quantity="1" unit="character"/>Ο<gap reason="lost" extent="unknown" unit="character"/>
akko0025.xml: <p>Μ</p>
beth0071.xml: <unclear>Σ</unclear>Α </p>
beth0072.xml: <supplied reason="lost">Σ</supplied>
beth0078.xml: <surplus>ΜΑ</surplus>
beth0085.xml: <supplied reason="lost">Σ</supplied>
beth0156.xml: <orig>ΕΑΡΝ</orig><lb/><orig>ΙΝ</orig>
beth0181.xml: <orig>ΑΙΕ</orig><lb/><orig>ΝΗ</orig>
beth0214.xml: <p>ΠΑΝ<lb/> ΠΕΙΝ </p>
bshe0135.xml: <unclear>Π</unclear>
caes0042.xml: <orig>ΓΕΜΗΝΙΣΤΟ</orig>
caes0151.xml: <supplied reason="undefined">Μ</supplied>
caes0155.xml: Ν<lb/>
caes0200.xml: <supplied reason="undefined">Ν</supplied>
caes0222.xml: <unclear>Υ</unclear>
caes0232.xml: <gap reason="illegible" quantity="2" unit="characters"/>Δ</supplied>
caes0240.xml: <gap reason="illegible" quantity="3" unit="character"/>Α<lb/>
caes0284.xml: <lb/>Τ<unclear/>
caes0298.xml: <unclear>Ι</unclear>
caes0302.xml: <unclear>Κ</unclear>Ο <unclear/>
caes0308.xml: <unclear>Τ</unclear>
caes0314.xml: <supplied reason="undefined">Ε</supplied>
caes0319.xml: <unclear/>Κ</p>
caes0333.xml: <unclear>Π</unclear>
caes0336.xml: <unclear>Χ</unclear>ρυσ<gap reason="illegible" quantity="2" unit="character"/>
caes0375.xml: <unclear>Α</unclear>
caes0377.xml: <p>ΘΑΑ<lb/>
caes0379.xml: <p>Ε<unclear>Τ</unclear>ΕΛ<unclear/>
caes0382.xml: <gap reason="illegible" quantity="2" unit="character"/>ΟΜΝΙΝ<unclear/>
caes0394.xml: <unclear>Α</unclear>Τ<gap reason="illegible" quantity="2" unit="character"/>
caes0524.xml: </choice><unclear>Ο</unclear></p>
caes0527.xml: <p><gap reason="lost" extent="unknown" unit="character"/>ΟΥ<lb/><gap reason="lost"
caes0661.xml: <p>Α</p>
caes0670.xml: <p><gap reason="lost" extent="unknown" unit="character"/>Ο<unclear>Ι</unclear>Π
dabb0001.xml: <foreign xml:lang="grc">ΡΟ</foreign>
deir0001.xml: <unclear>Δ</unclear>ο<supplied reason="lost">μ</supplied>
edda0001.xml: <abbr>Κ</abbr><ex>ύρι</ex>
egal0001.xml: <lb/>Α<g type="cross"/>Ω<lb/>
lus0005.xml: <gap reason="illegible"/>ΑΙΕΝΕ<unclear>Τ</unclear>
elus0009.xml: <gap reason="illegible" quantity="3" unit="character"/>Κ<gap reason="illegible" quantity="4" unit="character"/>Ρ<lb/>
elus0017.xml: <p>Μ<unclear>Υ</unclear>. <lb/>
elus0024.xml: <gap reason="illegible"/>ΟΝ<gap reason="illegible"/>
elus0028.xml: <unclear/>Ι Β<unclear>ΙΙ</unclear> +</p>
elus0029.xml: <gap reason="illegible"/>Κ<unclear/>
elus0032.xml: <abbr>Ζ<supplied reason="undefined">ο</supplied>ν<unclear>α</unclear>
elus0034.xml: <unclear>Α</unclear>
elus0039.xml: <orig>Σ</orig>
elus0042.xml: <unclear/>Ν<unclear>Θ</unclear>
elus0046.xml: <lb/>ΝΟ<unclear/>
elus0047.xml: <p>ΡΩ<lb/>
elus0048.xml: <p>ΤΟΥ</p>
elus0052.xml: <p>+ <unclear>Ι</unclear>ΕΛΑ</p>
elus0055.xml: <lb/>ΝΟ<unclear>Σ</unclear>
elus0058.xml: <supplied reason="undefined">Μ</supplied>ενᾶ <lb/>
elus0060.xml: <lb/>Ν<gap reason="illegible"/>
elus0062.xml: <lb/>ΚΑ<gap reason="illegible"/>
elus0064.xml: <lb/>Ν<unclear>Ι</unclear>
elus0065.xml: <unclear>Γ</unclear>ΙΑ<gap reason="illegible"/>
elus0070.xml: <lb/> Ν<unclear>Δ</unclear>
elus0071.xml: <p>ΑΖΟΝ</p>
elus0072.xml: <lb/>ΝΑ<unclear/>
elus0074.xml: <unclear/>ΝΙ</p>
elus0076.xml: <gap reason="illegible"/>Α<gap reason="illegible"/>
elus0077.xml: <unclear/>ΛΟ</p>
elus0079.xml: <gap reason="illegible"/>ΑΡΙ<gap reason="illegible"/>
elus0081.xml: <unclear/>Ν<gap reason="illegible"/>
elus0086.xml: <unclear/>ΕΥ<unclear/>
elus0088.xml: <lb/>ΚΑΙΤ</p>
elus0089.xml: <lb/>ΦΩΖΟΝ<unclear/>
elus0091.xml: <gap reason="illegible"/>Σ</p>
elus0092.xml: <unclear>Σ</unclear>Ι</p>
elus0094.xml: <unclear>ΕΜ</unclear>ΟΣ</p>
elus0095.xml: <p>+<unclear>ΟΡ</unclear>
elus0096.xml: <gap reason="illegible"/>ΒΑ<unclear>Ρ</unclear>
elus0099.xml: <p>ΣΩ<gap reason="illegible"/>
elus0102.xml: <unclear/>ΥΦΣΙΔ</p>
elus0104.xml: <lb/>ΤΙ<unclear/>ΡΛ<lb/>
elus0105.xml: <p>ΕΡ Ω<unclear>Ν</unclear>
fakh0004.xml: <unclear>Μ</unclear>
fein0011.xml: <orig>ΒΑ</orig>
fiqg0013.xml: <orig>Α</orig>
geze0001.xml: <foreign xml:lang="grc">ΑΛΚΙΟ<supplied reason="undefined">Υ</supplied>
geze0002.xml: <foreign xml:lang="grc">ΑΛΚΙΟΥ</foreign> תחמגזר</p>
geze0003.xml: <foreign xml:lang="grc">ΑΛΚΙΟΥ</foreign> תחמגזר</p>
geze0004.xml: <foreign xml:lang="grc">ΑΛΚΙΟΥ</foreign> תחמגזר</p>
geze0005.xml: <foreign xml:lang="grc">ΑΛΚΙΟΥ</foreign> תחמגזר</p>
geze0006.xml: <foreign xml:lang="grc">ΑΛΚΙΟΥ</foreign> תחמגזר</p>
geze0007.xml: <foreign xml:lang="grc">ΑΛΚΙΟΥ</foreign> תחמגזר</p>
geze0008.xml: <foreign xml:lang="grc">ΑΛ<unclear>ΚΙΟ</unclear>Υ</foreign>
haif0017.xml: <lb/>ΣΤΩΜΗ<lb/>ΛΙΕΝΟΝΤΙ ὁ τόπος</p>
hamm0027.xml: <unclear>Ι</unclear>ΑΧ<unclear>Θ</unclear>
jeri0006.xml: <p>ΑΒΓΔ<lb/>ΕΖ<lb/>ΗΘ<lb/>Φ
jeru0016.xml: <orig>ΜΑ<supplied reason="lost" cert="low">ΡΙΑΜΗ</supplied>
jeru0117.xml: <p>ΘΗ</p>
jeru0121.xml: <p>ΙΦΚΧ</p>
jeru0122.xml: <p>ΧΨ<lb/>ΑΒ</p>
jeru0130.xml: <supplied reason="undefined">Σ</supplied>α<unclear>ο</unclear>ύλου<lb/>
jeru0174.xml: <supplied reason="undefined">Ν</supplied>ιγερ
jeru0183.xml: <unclear>Κ</unclear>·<unclear>Ν</unclear>ΙΦ
jeru0185.xml: <foreign xml:lang="grc">Τ</foreign>
jeru0336.xml: <p>ΕΠΙ<lb/>LΕ ΣΤΡΑ<supplied reason="lost">ΤΟΥ</supplied>
jeru0337.xml: <p>ΔΙΟΝΥϹΙ<supplied reason="lost">Ο</supplied>Υ</p>
jeru0338.xml: <p>LΕ ΜϹΗ<supplied reason="omitted">Ι</supplied>
jeru0407.xml: <p>ΗΔ</p>
jeru0429.xml: <p>Υ<lb/>
jeru0438.xml: <unclear>Λ</unclear>
juei0003.xml: <abbr>Κ</abbr><ex>ύρι</ex>
kafr0001.xml: <gap reason="illegible"/>ΣΑΤΑΔ<gap reason="illegible"/>
kede0001.xml: <orig>ΘΙΝ</orig>Σε<lb break="no"/>πτίμι<lb break="no"/>ος <expan>
khis0071.xml: <abbr>Α</abbr>
mare0212.xml: <lb/>Lϝ<unclear>Μ</unclear>Ρ</p>
masa0282.xml: <lb/>Β</p>
masa0283.xml: <lb/>Β</p>
masa0285.xml: <p xml:lang="heb">Α
masa0301.xml: <g ref="#PHOENICIAN-LETTER-BET"/>Θ</p>
masa0302.xml: <foreign xml:lang="grc">Α</foreign>
masa0313.xml: <foreign xml:lang="grc">Α</foreign>
masa0329.xml: <foreign xml:lang="grc">Α</foreign>
masa0341.xml: <foreign xml:lang="grc">Α</foreign>
masa0351.xml: <foreign xml:lang="grc">Β</foreign>
masa0595.xml: <foreign xml:lang="grc">Ρ
masa0778.xml: <hi rend="supraline">Β</hi>
masa0795.xml: <num value="3">Γ</num>
masa0796.xml: <lb/>Καίσαρι <num value="8">Η</num>
masa0826.xml: <unclear>Δ</unclear>
masa0922.xml: <unclear>Θ</unclear>
masa0926.xml: <gap reason="lost" unit="character" extent="unknown"/>ΠΑΤΡΙΚΟΝ<lb/>
rafi0001.xml: <unclear>Χ</unclear>
sena0005.xml: <orig>ΑΝ</orig>
sena0006.xml: <gap reason="illegible"/>Η <unclear>Α</unclear> Υ<unclear>Τ</unclear>
sena0007.xml: <unclear>Σ</unclear>
sepp0015.xml: </supplied>ἐ­<space dim="horizontal"/>or<gap reason="illegible" quantity="9" unit="character"/>Ε<lb/>
shap0001.xml: <orig>ΜΑΡΑΤ</orig>
sina0108.xml: <foreign xml:lang="grc">ΖΕΟΣ</foreign>
sina0110.xml: <p>ΑΒΒΑΝΟΣ</p>
tala0001.xml: <unclear>Κ</unclear>άδμου <lb/>
tala0002.xml: <p>θάρσι Δο<lb/>μεττιά<lb/>ς <unclear>Α</unclear>βω<lb/>νου <expan>
tdor0004.xml: <abbr>Κ</abbr><ex>όιντον</ex>
ulat0007.xml: <abbr>Σ<supplied reason="lost">ε</supplied>β</abbr><ex>αστοὶ</ex>
ulat0013.xml: <supplied reason="undefined">Μ</supplied>ιγη<supplied reason="undefined">ραμ<lb/>ης</supplied>
yavn0001.xml: <gap reason="lost"/> ΔΕΧΑΤ<gap reason="lost"/>ΤΩΝΓΕΝ<gap reason="lost"/>
zoor0083.xml: <unclear>ΑΝΗΣΑΤΟΥ</unclear> ἐνάτου <unclear>ΑΝΑ</unclear>
zoor0087.xml: <gap reason="illegible"/> Ο<unclear>Τ</unclear>
zoor0102.xml: <p>+ <unclear>Ε</unclear>
zoor0106.xml: <corr>Κ</corr>
zoor0116.xml: <unclear>Χ</unclear>
zoor0127.xml: <unclear>Α</unclear>
zoor0143.xml: <supplied reason="undefined">Μ</supplied>
zoor0176.xml: <supplied reason="undefined">Ε</supplied>
zoor0219.xml: <supplied reason="lost">Μ</supplied>
zoor0221.xml: <unclear>Μ</unclear>
zoor0243.xml: <supplied reason="undefined">Κ</supplied>
zoor0246.xml: <abbr>Θ</abbr><ex>εό</ex>
zoor0247.xml: <sic>Σ<unclear>αββ</unclear>ίννης</sic>
zoor0254.xml: <abbr>Χ</abbr><ex>ριστο</ex>
zoor0255.xml: <p>Ε<supplied reason="undefined">ἶς
zoor0257.xml: <orig>Μ<unclear>ν</unclear>
zoor0273.xml: <unclear>ΕΝΟ</unclear>Σ <gap reason="illegible"/>
zoor0296.xml: <lb/> ΟΣΙ πάντες <gap reason="illegible"/>Ε<lb/> ἔχοντες<gap reason="illegible"/> τ<lb/>οῦ
zoor0317.xml: <sic>Θ<lb/>
zoor0319.xml: <gap reason="illegible"/>Θ<lb/>
zoor0320.xml: <lb/> Μ<gap reason="illegible"/>Ο<lb/> ΠΑ<gap reason="illegible"/>ΚΑΓ<lb/>
zoor0337.xml: <supplied reason="undefined">Π</supplied>
zoor0384.xml: <lb/>Α<gap reason="lost" quantity="8" unit="character" precision="low"/>
```

**Comment by emylonas** [comment]
Same issue as and resolved together with #112. All set. closing.
